src/app/Table.elm

```diff
@@ -0,0 +1,29 @@
+module Table exposing (view, init, Msg)
```

Comment:
The `tableView` function is the same function as `Table.view`, just lifted into the upper `Msg` type. This keeps them separated.